core/domain/state_domain.py

```diff
@@ -104,6 +107,50 @@ def from_dict(cls, answer_group_dict):
             answer_group_dict['tagged_skill_misconception_id']
         )
 
+    @classmethod
+    def to_proto(cls, answer_group):
```

Comment:
Same thing, why do you need to pass args here? Should be using self, similar to to_dict().

Reply:
Done